phlos.go

```diff
@@ -0,0 +1,73 @@
+package plivo
+
+const nodeType = "multi_party_call"
+
+type Phlo struct {
+	BaseResource
+
+	PhloId    string `json:"phlo_id" url:"phlo_id"`
+	Name      string `json:"name" url:"name"`
+	CreatedOn string `json:"created_on" url:"created_on"`
+}
+
+type Phlos struct {
+	BaseResourceInterface
+}
+
+func NewPhlos(client *PhloClient) (phlos *Phlos) {
+	phlos = &Phlos{}
+	phlos.client = client
+	//phlos.resourceType = Phlo // TODO: how to set this?
+
+	return
+}
+
+type PhloRun struct {
+	ApiID     string `json:"api_id" url:"api_id"`
+	PhloRunID string `json:"phlo_run_id" url:"phlo_run_id"`
+	PhloID    string `json:"phlo_id" url:"phlo_id"`
+	Message   string `json:"message" url:"message"`
+}
+
+func (self *Phlos) Get(phloId string) (response *Phlo, err error) {
```

Comment:
api_id is now added to the GetPHLO API. Update the code, struct, tests and fixtures to include that.

Reply:
Made necessary changes.